Formula/carla.rb

```diff
@@ -0,0 +1,27 @@
+class Carla < Formula
+  desc "Audio plugin host supporting LADSPA, LV2, VST2/3, SF2 and more"
+  homepage "https://kxstudio.linuxaudio.org/Applications:Carla"
+  url "https://github.com/falkTX/carla", :using => :git, :tag => "v1.9.9", :revision => "c03571a9ef95ac0e9564b95347f5de819aa7fb54"
```

Comment:
```ruby
url "https://github.com/falkTX/carla.git",
    :tag => "v1.9.9",
    :revision => "c03571a9ef95ac0e9564b95347f5de819aa7fb54"
```

I'm assuming there's a reason it has to be built from git instead of from the archive tarball?

Reply:
No, there is no reason, this was [choses at random from the example](https://github.com/syhw/homebrew/blob/master/Library/Contributions/example-formula.rb#L27).  Your comment suggests tarball is preferred over git, this will be corrected.